README.md

```diff
@@ -124,7 +124,76 @@ Run `ng g component component-name` to generate a new component. You can also us
 ## Build
 
 Optionally override the webservice version using `npm config set dockstore-ui2:webservice_version ${WEBSERVICE_VERSION}`
-Run `npm run build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `npm run build.prod` for a production build.
+Run `npm run build` to build the project. The build artifacts will be stored in the `dist/` directory. 
+
+### Production Build
+
+For a production build, you will need to install Nginx. 
+Replace your nginx.conf file (location depends on your installation) with this template, filling in the two paths:
+```
+events {
```

Comment:
Sorry, this was my suggestion -- I had run the Angular prod build like that before, and thought it was easier to just use a new one for running on localhost. For the one in you have to run `install_bootstrap` which I don't think works on a Mac,  assumes that the webservice is accessible at http://webservice, etc.. This is a fallback for a less complex setup for a local build.

Reply:
@denis-yuen Was my argument at all convincing or do you still feel we shouldn't have this?

Reply:
Oh sorry, forgot. 
I think there's a couple issues, but I may not fully understand. 
Assuming the idea here is to create an nginx that can run locally for a developer but pass the audit checks

1) This configuration will/may have(?) diverge from compose_setup
2) It's not clear to me what needs to be taken from this and moved over the compose_setup and it is compose_setup which "counts" since that is what is running on staging.dockstore.org and dockstore.org
3) If the problem with running compose_setup is that is doesn't run on a Mac, we should fix that rather than have this workaround. 

That said, I'm fine with this as a temporary workaround, with the knowledge that we will need a follow-up ticket to get this into the "real" nginx config in compose_setup.

Reply:
Do you run compose_setup with the web service running via the IDE's debugger?  I would think that would be tricky, but maybe I don't know docker-compose well enough.

It would be worth the teams comparing our local dev setups at some point; we might be doing some things wrong or inefficiently on our side.